perun-core/src/main/java/cz/metacentrum/perun/core/entry/VosManagerEntry.java

```diff
@@ -380,6 +380,7 @@ public void removeAdmin(PerunSession sess, Vo vo, Group group) throws PrivilegeE
 		//Role can be only supported one (TopGroupCreator, VoAdmin or VoObserver)
 		if(!role.equals(Role.TOPGROUPCREATOR) &&
 						!role.equals(Role.VOADMIN) &&
+						!role.equals(Role.SPONSOR) &&
 						!role.equals(Role.VOOBSERVER)) {
 			throw new RoleNotSupportedException("Supported roles are VoAdmin, VoObserver and TopGroupCreator.", role);
```

Comment:
You should change the message of the exception too.

Reply:
Done.